### PR TITLE
singularity v3.6.0

### PR DESCRIPTION
**Pull request by davidchristiany**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -32,6 +32,26 @@
...
+    - build-essential
+    - uuid-dev
+    - libgpgme-dev
+    - squashfs-tools
+    - libseccomp-dev
+    - wget
+    - pkg-config
+    - git
+    - cryptsetup-bin
+    state: latest
```
Comment (lecorguille) [inline tasks/main.yml]: Pleas look here:  https://github.com/abims-sbr/ansible-singularity/blob/master/vars/Debian.yml

File: tasks/main.yml [hunk 2]
```diff
@@ -32,6 +32,26 @@
   changed_when: False
   failed_when: False
 
+- name: singularity | remove older Go installation
+  file:
+    path: '/opt/go/{{ item }}'
+    state: absent
+  with_items: "{{ old_go_versions }}"
```
Comment (lecorguille) [inline tasks/main.yml]: You can do better than listing all the possible old version 😛 
Maybe by:
 - listing and put the result in a variable using `register`
 - then filter to exclude the current one with https://jinja.palletsprojects.com/en/2.10.x/templates/#filters

File: tasks/main.yml [hunk 3]
```diff
@@ -32,6 +32,26 @@
   changed_when: False
   failed_when: False
 
+- name: singularity | remove older Go installation
```
Comment (lecorguille) [inline tasks/main.yml]: I should be interesting to add a little switch as a variable if some infra want to keep for some reason all the version of GO ?

**Review by lecorguille — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (davidchristiany) [reply tasks/main.yml]: Ok, I'll update the list

Comment (lecorguille) [reply tasks/main.yml]: There is also on for redhat. You need to find the equivalents

File: tasks/main.yml (hunk 2 above)
Comment (davidchristiany) [reply tasks/main.yml]: Where do I get the list of go version ? I can do it from conda but it's better if it doesn't need conda

File: tasks/main.yml (hunk 3 above)
Comment (davidchristiany) [reply tasks/main.yml]: Done

File: tasks/main.yml (hunk 2 above)
Comment (lecorguille) [reply tasks/main.yml]: With [find_module](https://docs.ansible.com/ansible/latest/modules/find_module.html), you can list all the GO versions installed on the filesystem.

Comment (davidchristiany) [reply tasks/main.yml]: I didn't need to filter, I used regex to exclude the current go installation

File: defaults/main.yml [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
 
 singularity_conf_path: "/usr/local/etc/singularity/singularity.conf"
 
 bind_paths: []
 # example
 # - /shared/home/
 # - /shared/project/
+
+#go variables
+only_one_go_version: true
```
Comment (lecorguille) [inline defaults/main.yml]: Can you switch it to false to not change the previous behaviour?

**Review by lecorguille — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 4 above)
Comment (davidchristiany) [reply defaults/main.yml]: sure

Comment (davidchristiany) [reply defaults/main.yml]: sure

**Review by lecorguille — APPROVED** [review]

**Pull request merged**